charts/bulk-scan-orchestrator/values.template.yaml

```diff
@@ -0,0 +1,32 @@
+java:
+  applicationPort: 8582
+  readinessPath: '/health'
+  livenessPath: '/health'
+  secrets:
+    SB_CONN_STRING:
+      secretRef: servicebus-secret-namespace-${SERVICE_NAME}
+      key: connectionString
+  environment:
+    S2S_SECRET: "${S2S_SECRET}"
+    IDAM_CLIENT_SECRET: "${IDAM_CLIENT_SECRET}"
+    IDAM_USER_NAME: "${IDAM_USER_NAME}"
+    IDAM_USER_PASSWORD: "${IDAM_USER_PASSWORD}"
+    IDAM_USERS_BULKSCAN_USERNAME: "${IDAM_USER_NAME}"
```

Comment:
is the next PR adding properties volume and removing this 😉

Reply:
Yes, I was waiting for that part to be tested ... no point in having to revert it